src/Components/test/testassets/BasicTestApp/EventBubblingComponent.razor

```diff
@@ -1,15 +1,67 @@
 <h3 id="event-bubbling">Bubbling standard event</h3>
 
+@* Temporarily hard-coding the internal names - this will be replaced once the Razor compiler supports @onevent:stopPropagation and @onevent:preventDefault *@
 <div @onclick="@(() => LogEvent("parent onclick"))">
-    <button id="button-with-onclick" @onclick="@(() => LogEvent("target onclick"))">Button with onclick handler</button>
-    <button id="button-without-onclick" >Button without onclick handler</button>
+    @* This element shows you can stop propagation even without necessarily also handling the event *@
+    <div __internal_stopPropagation_onclick="@intermediateStopPropagation">
+        <button id="button-with-onclick" @onclick="@(() => LogEvent("target onclick"))" __internal_stopPropagation_onclick="@targetStopPropagation">
+            Button with onclick handler
+        </button>
+        <button id="button-without-onclick">
+            Button without onclick handler
+        </button>
+    </div>
 </div>
 
+<fieldset>
+    <legend>Options</legend>
+    <label>
+        <input id="intermediate-stop-propagation" type="checkbox" @bind="intermediateStopPropagation" />
+        Stop propagation on intermediate element
+    </label>
+    <label>
+        <input id="target-stop-propagation" type="checkbox" @bind="targetStopPropagation" />
+        Stop propagation on target element
+    </label>
+</fieldset>
+
+<h3>PreventDefault</h3>
+
+<div __internal_preventDefault_onclick="@ancestorPreventDefault">
+    <p>
+        <label>
+            <input type="checkbox" id="checkbox-with-preventDefault-true" __internal_preventDefault_onclick @onclick="@(() => LogEvent("Checkbox click"))" @onchange="@(() => LogEvent("Checkbox change"))" />
```

Comment:
Scrolling through these - it looks like this is the only coverage of the minimized form. @ajaybhargavb - let's make sure we preserve the semantics of this.